sklearn/ensemble/tests/test_forest.py

```diff
@@ -204,6 +204,22 @@ def check_probability(name):
                                   np.exp(clf.predict_log_proba(iris.data)))
 
 
+@pytest.mark.parametrize('name', FOREST_ESTIMATORS)
+def test_warning_raised_with_deprecated_n_estimator(name):
+    ForestClassifier = FOREST_CLASSIFIERS[name]
+    clf = ForestClassifier(n_estimators=10, random_state=1, max_features=1,
+                           max_depth=1)
+    assert_warns(DeprecationWarning, func=clf.fit, X=iris.data, y=iris.target)
+
+
+@pytest.mark.parametrize('name', FOREST_ESTIMATORS)
+def test_no_warning_raised_with_correct_n_estimator(name):
+    ForestClassifier = FOREST_CLASSIFIERS[name]
+    clf = ForestClassifier(n_estimators=100, random_state=1, max_features=1,
+                           max_depth=1)
+    assert_warns(DeprecationWarning, func=clf.fit, X=iris.data, y=iris.target)
```

Comment:
shouldn't this be `assert_no_warnings`?

Reply:
Totally missed that. 

I've done well to fix it in my latest commit.